Formula/tomcat@8.0.rb

```diff
@@ -3,16 +3,15 @@ class TomcatAT80 < Formula
   homepage "https://tomcat.apache.org/"
 
   stable do
-    url "https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-8/v8.0.42/bin/apache-tomcat-8.0.42.tar.gz"
-    sha256 "be876f955657791a480c167ea126663f56aa908da5c354bd4b3d60b3aa5c6e1e"
+    url "http://apache.mirrors.pair.com/tomcat/tomcat-8/v8.0.43/bin/apache-tomcat-8.0.43.tar.gz"
```

Comment:
That was an intentional change. This mirror-choosing script does not seem to work for me, it returns 22KB html and not a 8MB binary:

```sh
tbasanov@tbasanov:~/Downloads⟫ wget 'https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-8/v8.0.42/bin/apache-tomcat-8.0.42.tar.gz'     
--2017-04-07 13:11:03--  https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-8/v8.0.42/bin/apache-tomcat-8.0.42.tar.gz
Resolving www.apache.org... 88.198.26.2, 140.211.11.105, 2a01:4f8:130:2192::2
Connecting to www.apache.org|88.198.26.2|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘closer.cgi?path=tomcat%2Ftomcat-8%2Fv8.0.42%2Fbin%2Fapache-tomcat-8.0.42.tar.gz.1’

closer.cgi?path=tomcat%2Ftomcat-     [ <=>                                                     ]  22.33K   131KB/s    in 0.2s

2017-04-07 13:11:04 (131 KB/s) - ‘closer.cgi?path=tomcat%2Ftomcat-8%2Fv8.0.42%2Fbin%2Fapache-tomcat-8.0.42.tar.gz.1’ saved [22861]

tbasanov@tbasanov:~/Downloads⟫ wget 'https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-8/v8.0.43/bin/apache-tomcat-8.0.43.tar.gz'
--2017-04-07 13:11:16--  https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-8/v8.0.43/bin/apache-tomcat-8.0.43.tar.gz
Resolving www.apache.org... 88.198.26.2, 140.211.11.105, 2a01:4f8:130:2192::2
Connecting to www.apache.org|88.198.26.2|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘closer.cgi?path=tomcat%2Ftomcat-8%2Fv8.0.43%2Fbin%2Fapache-tomcat-8.0.43.tar.gz’

closer.cgi?path=tomcat%2Ftomcat-     [ <=>                                                     ]  22.36K   133KB/s    in 0.2s

2017-04-07 13:11:17 (133 KB/s) - ‘closer.cgi?path=tomcat%2Ftomcat-8%2Fv8.0.43%2Fbin%2Fapache-tomcat-8.0.43.tar.gz’ saved [22897]
```

I can can switch to using apache org directly with URLs like this: http://archive.apache.org/dist/tomcat/tomcat-8/v8.0.43/bin/apache-tomcat-8.0.43.tar.gz

Not sure if that's the right CGI script to look at, but looks like it's html/json/xml only: http://svn.apache.org/viewvc/infrastructure/site/trunk/content/dyn/mirrors/mirrors.cgi?view=markup